pragma-node/src/infra/repositories/entry_repository.rs

```diff
@@ -782,6 +782,8 @@ fn get_table_name_from_type(entry_type: DataType) -> &'static str {
     }
 }
 
+const EXCLUDED_PUBLISHER: &str = "PRAGMA";
```

Comment:
Can we add a smol comment explaining *why* we exclude the Pragma publisher? So we don't forgot in later maintenances